PyRDF/RDataFrame.py

```diff
@@ -183,6 +183,29 @@ def get_inputfiles(self):
         # RDataFrame may have been created with no input files
         return None
 
+    def get_input_friend_treenames_and_filenames(self):
```

Comment:
Also I would use a shorter name, maybe the same as in the ROOT implementation: `get_friend_info`.